### PR TITLE
Remove incorrect log message when an issuer is an empty string

### LINKED ISSUE #1673: IssuerValidator logs incorrect message when an issuer is an empty string
When we iterate through ValidIssuers on the TokenValidationParameters and discover an issuer that is an empty string we throw the following incorrect exception:

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/48d02044accd936ec1749ca8f5d349a5a9c5e6df/src/Microsoft.IdentityModel.Tokens/Validators.cs#L242
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/48d02044accd936ec1749ca8f5d349a5a9c5e6df/src/Microsoft.IdentityModel.Tokens/LogMessages.cs#L71

### PR DESCRIPTION
**Pull request by ciaozhang**
Close #1673

### CODE REVIEW

File: src/Microsoft.IdentityModel.Tokens/Validators.cs [hunk 1]
```diff
@@ -238,10 +238,7 @@ public static string ValidateIssuer(string issuer, SecurityToken securityToken,
                 foreach (string str in validationParameters.ValidIssuers)
                 {
                     if (string.IsNullOrEmpty(str))
-                    {
-                        LogHelper.LogInformation(LogMessages.IDX10235);
```
Comment (TimHannMSFT) [inline src/Microsoft.IdentityModel.Tokens/Validators.cs:242]: Should we still have something logged in this case (this string just seems like the wrong string).

This does seem like something that indicates that something odd is happening, if there are empty strings in the ValidIssuers collection something has gone awry correct?

Comment (ciaozhang) [reply src/Microsoft.IdentityModel.Tokens/Validators.cs:242]: IMHO, we just need the valid issuer is logged here.

Comment (jennyf19) [reply src/Microsoft.IdentityModel.Tokens/Validators.cs:242]: Looks like that error message gets returned on line 216. If this is actionable for the developer (ex. they need to set a property, etc..), if so, I would definitely return something that says what the action should be. If not, it's probably fine to not log anything, or just log that the issuer was null or empty for debugging purposes.

Comment (brentschmaltz) [reply src/Microsoft.IdentityModel.Tokens/Validators.cs:242]: @ciaozhang i agree with Tim having a null or empty string is unusual.
The log message should say something like: "one of the issuers in TokenValidationParameters.ValidIssuers was null or the empty string."

**Pull request closed without merging**